docs/apps/features.md

```diff
@@ -0,0 +1,79 @@
+# Features
+
+The django-ansible-base `features` app provides a way to have new features
+in development that an end use can opt into.
+
+## Consumer Docs
+
+To make use of `features` in your application, first add
+`ansible_base.features` to your `INSTALLED_APPS`.
+
+Now in your code you can:
+
+```python
+from ansible_base.features import  feature_enabled
+
+if feature_enabled('some_feature_short_name')
+    ... feature code here ...
+```
+
+### Caching
+
+The feature enabled/disabled flags are stored in the database but can be cached
+out by setting: ANSIBLE_BASE_FEATURE_CACHE=<cache name>
+
+### Registering Features
+
+Once the features app is installed we have to tell it to register features. To do this add a file called `dab_features.py` at the root of your application. This file should have a single variable called `FEATURES` in it that is an array of dicts describing features:
+```
+FEATURES = [
+    {
+        'short_name': 'SLO',
+        'name': 'Single Logout',
+        'description': 'Add single logout to SAML and OIDC authenticators',
+        'status': 'a',
+        'requires_restart': True,
+    }
+]
+```
+
+`short_name` str, an easy to reference name that will be used as an identifier inside the code
+`name` str, the display name for the feature (end user facing)
+`description` str, a brief description of the feature
+`status` one of ['a', 'b'], the status of the feature (Alpha or Beta, see Feature Status)
```

Comment:
so we should also have teach preview and GA? Because there will be normal supported features here?